src/serde_arrow_type.erl

```diff
@@ -48,27 +50,37 @@
 %% Any floating point number in Apache Arrow. Includes `Float 16', `Float 32'
 %% and `Float 64'.
 
+-type bin() :: bin.
```

Comment:
As mentioned in the other PR, we should be more consistent here. Maybe the type should be `arrow_bin` with value of `bin`? Otherwise, why is it `arrow_bool()` with value of `arrow_boolean`?

Reply:
Yeah, it should be `arrow_bin`.